app/models/loader/types.rb

```diff
@@ -134,9 +134,27 @@ def verify; end
     end
 
     class Host < Record
-      def verify; end
       def_delegators :@policy_object, :restricted_to
 
+      def api_key_enabled?
+        annot = self.annotations&.[]("authn/api-key")
+        return annot unless annot.nil?
+
+        config = Rails.application.config.conjur_config.authn_api_key_default
+        return config unless config.nil?
+
+        true
```

Comment:
The config value should never be `nil`, as this same check is part of the config class itself. This can return the value directly without the additional condition check.

Reply:
I went on the safe side, but I will remove the check if you're sure it's not needed